packages/stats/gbstats/gbstats.py

```diff
@@ -741,6 +741,8 @@ def process_experiment_results(
                         # when using multi-period data, binomial is no longer iid and variance is wrong
                         if metric_settings_bandit.main_metric_type == "binomial":
                             metric_settings_bandit.main_metric_type = "count"
+                        if metric_settings_bandit.covariate_metric_type == "binomial":
+                            metric_settings_bandit.covariate_metric_type = "count"
```

Comment:
do we ever have to worry about the case where `main_metric_type` differs from `covariate_metric_type`?